src/App.tsx

```diff
@@ -1,16 +1,22 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [allMovies, setAllMovies] = useState(moviesFromServer);
+
   return (
     <div className="page">
       <div className="page-content">
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={allMovies} />
       </div>
       <div className="sidebar">
-        <NewMovie /* onAdd={(movie) => {}} */ />
+        <NewMovie onAdd={(movie) => {
+          setAllMovies([...allMovies, movie]);
+        }}
+        />
```

Comment:
it will be better to create separate `onAdd` function and move the code there
```suggestion
        <NewMovie onAdd={onAdd} />
```